getting-started/chat-thread.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Chat and thread"
+weight: 10
+---
+
+# Chat and thread
+
+
+### Basics
+
+In our system chat is divided into threads. Threads are standalone parts of chat and can contain events. In every chat multiple users (customers or agents) can participate and every user can have multiple chats at the same time.
```

Comment:
In our system chat is divided into threads -> In our system a chat (albo: each chat) is divided into threads

Threads are standalone parts of chat > Threads are standalone parts of a chat

In every chat multiple users (customers or agents) can participate and every user can have multiple chats at the same time -> Multiple users (customers or agents) can participate in a chat and every user can have multiple chats at the same time